src/scss/partials/_header.scss

```diff
@@ -98,7 +98,7 @@ $header--transparent-opacity        : 0.7;
         right: spacing();
         padding: spacing();
         position: absolute;
-        z-index: zIndex(high);
+        z-index: zIndex(high) - 1;
```

Comment:
Sorry gona be picky here. I don't think we should do this, sort of defeats the purpose of the mapper. We should either add a new value in the mapper or just add the value directly here. e.g. `z-index: 101;`

Reply:
@kevinrodrigues okay no trouble. I'll just add the value directly here as it _kinda_ needs to be just one below high :)